Library/Formula/wine.rb

```diff
@@ -5,7 +5,6 @@
 class Wine < Formula
   desc "Wine Is Not an Emulator"
   homepage "https://www.winehq.org/"
-  revision 1
```

Comment:
Nonetheless it applies to both stable and devel versions, so the version for stable is:
```
https://homebrew.bintray.com/bottles/wine-1.6.2_1.el_capitan.bottle.tar.gz
```
If we remove the revision bump and then have to add it again for some issue later stable users won't get that revision because they'll already be on it.